packages/flutter/lib/src/material/text_selection_theme.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/cupertino.dart';
```

Comment:
Do you have to import this in order to reference `CupertinoThemeData` in the docs?  We usually try to keep Material and Cupertino independent of each other.

Reply:
I'll check on this one.